CardiacDeviceSimulator/CardiacDeviceSimulator.py

```diff
@@ -647,7 +649,7 @@ def setParameterNode(self, parameterNode):
       n = slicer.mrmlScene.CreateNodeByClass("vtkMRMLProceduralColorNode")
       n.UnRegister(None)
       n.SetName(slicer.mrmlScene.GenerateUniqueName("Radial compression (mm)"))
-      n.SetAttribute("Category", "CardiacDeviceSimulator")
+      n.SetAttribute("Category", "CardiacModelGenerator")
```

Comment:
I am not sure if it should be the same category as it was in the past

Reply:
You've done it right. We can set any name but it is more clear what it is for if the category name matches module name.

Reply:
Does that affect any of the already created models from the "old" module?

Reply:
It does not affect any old scenes. Category only defines how it shows up in the color node tree.